### PR TITLE
Create stub for resistor-color exercise

### PR DESCRIPTION
**Pull request by kytrinyx**
Does this test suite look reasonable to you? Is there anything you'd like to change before we implement a reference solution?

### CODE REVIEW

**Comment by lucaferranti** [comment]
@kytrinyx 

overall looks very good, a few comments.

- the configlet test was failing because `chapel/config.json` was using `title` instead of `name`. Not sure if it was a typo in the generator or if I'm using an outdated configlet

- (very pedantic, almost feel sorry for pointing it out :) ) in the `reference.chpl` and stub file the `// insert your solution here` only had 1 space indentation, generally chapel uses 2 (to be fair, I didn't find it written, but checking throw the chapel core modules it feels pretty consistent

File: config.json [hunk 1]
```diff
@@ -45,7 +45,7 @@
       {
         "uuid": "0b8f9871-7dd8-4f51-b8e3-be60c9297643",
         "slug": "resistor-color",
-        "title": "Resistor Color",
+        "name": "Resistor Color",
```
Comment (kytrinyx) [inline config.json:48]: D'oh. I will fix my script.

File: exercises/practice/resistor-color/.meta/reference.chpl [hunk 2]
```diff
@@ -1,3 +1,11 @@
 module ResistorColor {
- // implement reference solution
+  // implement reference solution
```
Comment (kytrinyx) [inline exercises/practice/resistor-color/.meta/reference.chpl]: Let's delete the comment, too.

File: exercises/practice/resistor-color/src/resistor-color.chpl [hunk 3]
```diff
@@ -1,3 +1,3 @@
 module ResistorColor {
- // write your solution here
+  // write your solution here
```
Comment (kytrinyx) [inline exercises/practice/resistor-color/src/resistor-color.chpl:2]: Let me fix the template for this.

**Comment by kytrinyx** [comment]
I'm also very nitpicky—those things were just complete oversights on my part! I've fixed them now :)

The configlet thing is totally me also. I was looking at the `metadata.toml` for an exercise that had a `title` field in the problem-specifications, but which we call `name` within the track config.

**Comment by kytrinyx** [comment]
I think this is good to go!

**Review by lucaferranti — APPROVED** [review]

**Pull request merged**